apps/emqx_utils/src/bpapi/emqx_bpapi_trans.erl

```diff
@@ -87,7 +87,7 @@ mk_meta_fun(#s{api = API, version = Vsn, targets = Targets}) ->
     Line = 0,
     Calls = [{From, To} || {call, From, To} <- Targets],
     Casts = [{From, To} || {cast, From, To} <- Targets],
-    Ret = typerefl_quote:const(Line, #{
+    Ret = erlang_qq:const(Line, #{
```

Comment:
Quasiquotation parse transform has been extracted from typerefl to a new `erlang_qq` library. But they are functionally the same.